docs/design.md

```diff
@@ -43,8 +43,9 @@ The dialect determines what SQL is generated for various Tableau actions.
 
 Choosing the right dialect is a critical part of writing a connector. For example:
 
-- If you set <span style="font-family: courier new">base</span> to "mysql_odbc" as the parent, you can skip configuring a dialect, and your connector will use the parent’s dialect.
```

Comment:
Can you invert the ordering of these to better show our recommendations. Complete is ideal, followed by inherit base, followed by superclass

Reply:
Done in the latest commit, thanks, I just committed your suggestions for other comments, thanks for writing them out for me.